.devcontainer/Dockerfile

```diff
@@ -320,7 +320,14 @@ RUN ldconfig -v
 
 
 ##### Install Python requirements
-RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0"
+RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0" "virtualenv==20.0.17"
+# use same virtualenv version that is installed in VM
+# TODO: install virtualenv with apt instead?
+
+### create virtualenv
+ARG PYTHON_VENV=/home/vscode/build/python
+
+RUN virtualenv --system-site-packages --python=/usr/bin/python3.8 $PYTHON_VENV
```

Comment:
If we use `/usr/bin/python3.8` in multiple places (also below), we should probably make it an `ARG` (or `ENV`, depending...).

Reply:
Couldn't find another use of this explicit string but put `lib/python3.8/gen` in a `GEN_DIR` variable. Hope that also helps.

Reply:
The target would be, that we cannot break the setup by forgetting to update some string somewhere.

Reply:
Got it! There also seems to be the possibility to reference environment variables in the devcontainer.json. That might help as well. Looking into it.

Reply:
I created an `ARG PYTHON_VERSION`. With that we have captured all the variable parts of the paths in variables I suppose (assuming that python will be installed via apt and the virtual environment is created with virtualenv) Let me know what you think.